morango/syncsession.py

```diff
@@ -95,11 +116,14 @@ def _request(self, endpoint, method="GET", lookup=None, data={}, params={}, user
             lookup = lookup + '/'
         url = urljoin(urljoin(self.base_url, endpoint), lookup)
         auth = (userargs, password) if userargs else None
-        resp = requests.request(method, url, json=data, params=params, auth=auth)
+        if gzip:
```

Comment:
Given the way things are structured, found it easiest to separate into 2 requests.